frontend/package.json

```
@@ -36,6 +36,7 @@
     "storybook:test": "test-storybook",
     "test:lint": "npm run test:lint:eslint && npm run test:lint:style && npm run test:lint:locales",
     "test:lintfix": "npm run test:lint:eslint -- --fix && npm run test:lint:style -- --fix && npm run test:lint:locales -- --fix",
+    "test:lintfix:all": "npm run test:lint:eslint -- --fix; npm run test:lint:style -- --fix; npm run test:lint:locales -- --fix",
```

These scripts are a little messy in our project. I vote for [npm-run-all](https://www.npmjs.com/package/npm-run-all) or a similar helper.

Then we could do:
```suggestion
    "lint": "run-p -l lint:* -- ",
```

And on the command line you can do:
```
npm run lint -- --fix
```

That should be easier.

---

Oh well, `npm-run-all` hasn't been touched since 2018. Maybe https://github.com/bcomnes/npm-run-all2 or whatever similar helper.